test/testdata/ingress/stargate-ingress.yaml

```diff
@@ -2,40 +2,40 @@
 apiVersion: traefik.containo.us/v1alpha1
 kind: IngressRoute
 metadata:
-  name: test-dc1-stargate-service-http-ingress # actual namespace will be kustomized
-  namespace: default # actual namespace will be kustomized
+  name: cluster1-dc1-stargate-service-http-ingress
+  namespace: k8ssandra
 spec:
   entryPoints:
-    - web-http
+    - web
   routes:
-    - match: Host(`stargate.127.0.0.1.nip.io`) && PathPrefix(`/v1/auth`)
+    - match: Host(`stargate.example.com`) && PathPrefix(`/v1/auth`)
```

Comment:
The changes in the Ingress yaml definitions are merely to make these files more illustrative. John asked me a while ago to always consider the yaml fixture files as self-documenting examples, so I chose more generic names for hosts, namespaces, service names, etc. The actual names don't matter, they will be kustomized before getting applied.